source/legacy_members_api.rst

```diff
@@ -208,3 +208,67 @@ On successful balance update:
       "created": "2014-08-08",
       "extra_data": {},
     }
+
+
+4. Get Merged Legacy Members
+====================================
+| URL endpoint: https://stamps.co.id/api/legacy/members/get-merged-members
+| Allowed Method: GET
+| Require Authentication: Yes
+
+A. Request
+-----------------------------
+You can get legacy members that have been merged to a stamps membership with this API.
```

Comment:
Allows you to examine legacy members that have been merged into a user's membership account.